RadarSDK/Radar.m

```diff
@@ -1151,6 +1152,41 @@ + (void)logResigningActive {
 }
 
 
+#pragma mark - Indoors
+
++ (void)doIndoorSurvey:(NSString *)placeLabel
+             forLength:(int)surveyLengthSeconds
+        isWhereAmIScan:(BOOL)isWhereAmIScan
+     completionHandler:(RadarIndoorsSurveyCompletionHandler)completionHandler {
```

Comment:
```suggestion
+ (void)startIndoorSurvey:(NSString *)placeLabel
             duration:(int)surveyLengthSeconds
     completionHandler:(RadarIndoorsSurveyCompletionHandler)completionHandler {
```
I'd suggest the above renaming (a bit more clean, idiomatic).


We don't need `isWhereAmIScan:(BOOL)isWhereAmIScan` because that's always true for a survey. And I think we should rename that downstream as `survey` or something of the like.